### ISSUE TITLE
Build DIGITS on Windows

### ISSUE THREAD

**Issue by JieZou1** [NVIDIA/DIGITS#910]
@IsaacYangSLA 

I have followed the steps described in  docs / BuildDigitsWindows.md to try run DIGITS on Windows 7, but I receive the following error message in the last step, running "python digits-devserver"

------------------------------------
Traceback (most recent call last):
  File "digits-devserver", line 31, in <module>
    from digits import config
  File "D:\digits\digits\config\__init__.py", line 7, in <module>
    from .current_config import config_value
  File "D:\digits\digits\config\current_config.py", line 4, in <module>
    from .caffe_option import CaffeOption
  File "D:\digits\digits\config\caffe_option.py", line 14, in <module>
    from digits.utils import parse_version
  File "D:\digits\digits\utils\__init__.py", line 158, in <module>
    from . import constants, image, time_filters, errors, forms, routing, auth
  File "D:\digits\digits\utils\image.py", line 15, in <module>
    import PIL.Image
ImportError: No module named PIL.Image
--------------------------------------

I am using Anaconda, the Python version is:

Python 2.7.11 |Anaconda 4.0.0 (64-bit)| (default, Feb 16 2016, 09:58:36) [MSC v. 1500 64 bit (AMD64)] on win32

I do have installed all the modules listed in the requirests.txt. I also have installed image module, which is not included in Anaconda.

Any ideas what goes wrong? Many thanks

**Comment by TimZaman** [comment]
Seems you should also download the Pillow package as you downloaded the other packages and then run
`python -m pip install Pillow-3.3.0-cp27-cp27m-win_amd64.whl`

**Comment by IsaacYangSLA** [comment]
Yes.  What Tim said was right.  Please install Pillow.  That package is the first one in requirements.txt.  I will check if my instruction does not cover that.  Thanks.

**Comment by JieZou1** [comment]
@TimZaman @IsaacYangSLA 

Many thanks, Tim and Isaac, previously, I have pillow 3.2.0-py27_0 installed. After upgrading the pillow to 3.2.0-py27_1, DIGITS does successfully run on my Win7. The version is  higher than that listed in the requirements.txt (Pillow>=2.3.0,<=3.1.2), but DIGITS does still seem running Okay. Many thanks indeed!

**Comment by IsaacYangSLA** [comment]
@JieZou1 , great to hear that.  I will close this issue soon.  However, please feel free to comment in case you encounter the same issue again.

**Comment by Chachay** [comment]
@IsaacYangSLA @JieZou1 

I got the similar problem. The issue must be from the part of requirement.txt in BuildDigitsWindows.md.

```
python -m pip install -r requirements.txt
```

This command makes all PIL file names into lower case with the latest pip script (8.1.2 on windows 10), which led small messes. 
E.g. PIL/Image.py into pil/image.py

In the end, I edited requirements.txt to remove "pillow" line
